src/jit/compiler.cpp

```diff
@@ -4428,11 +4428,12 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, CORJIT_F
     // Stash the current estimate of the function's size if necessary.
     if (verbose)
     {
-        compSizeEstimate = 0;
+        compSizeEstimate  = 0;
         compCycleEstimate = 0;
         for (BasicBlock* block = fgFirstBB; block != nullptr; block = block->bbNext)
         {
-            for (GenTreeStmt* statement = block->firstStmt(); statement != nullptr; statement = statement->getNextStmt())
+            for (GenTreeStmt* statement = block->firstStmt(); statement != nullptr;
+                 statement              = statement->getNextStmt())
             {
```

Comment:
I'm not a fan of this either - is there a way that I can improve it that jit-format will be happy with?

---
In reply to: [77255945](https://github.com/dotnet/coreclr/pull/7027#discussion_r77255945) [](ancestors = 77255945)